libs/charts/src/lib/components/chart-container/chart-container.component.ts

```diff
@@ -139,8 +139,10 @@ export interface IChartConfiguration {
   type?: string;
 
   data?: {
+    labels?: Array<unknown>
     datasets?: {
       label: string;
+      data: Array<unknown>
```

Comment:
Honestly not sure what this type should be, and the type checker doesn't seem to care

I can use `<T>` but I'd have to use that like everywhere in this file

Reply:
That'll work for now. I might need to go in a figure out why it's not caring about it